### PR TITLE
Ocs 311

### PR DESCRIPTION
**Pull request by cooktheryan**
Addition of links for ocs deployment for the openshift providers that would benefit from having OCS deployed.

@netzzer

### CODE REVIEW

File: install_config/configuring_aws.adoc [hunk 1]
```diff
@@ -313,3 +313,5 @@ Balancing (ELB) will not be able to create a load balancer.
 ====
 
 After tagging any resources, restart the master services on the master and the node service on all nodes. See the xref:aws-applying-configuration-changes[Applying Configuration Section].
+
+include::install_config/topics/red_hat_openshift_container_storage.adoc[leveloffset=+2]
```
Comment (kalexand-rh) [inline install_config/configuring_aws.adoc]: You need to add this line somewhere before line 317:

 :context: configuring-for-aws

File: install_config/topics/red_hat_openshift_container_storage.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Module included in the following assemblies:
+
+install_config/configuring_aws.adoc
+install_config/configuring_azure.adoc
+install_config/configuring_gce.adoc
+install_config/configuring_vmware.adoc
+////
+
+= Configuring OpenShift Container Platform for OpenShift Container Storage 
+Red Hat OpenShift Container Storage (RHOCS) is a provider of agnostic persistent storage for OpenShift Container Platform (OCP) on-premise and in hybrid clouds, As a Red Hat storage solution, RHOCS is completely integrated with OCP for deployment, management and monitoring regardless if it is installed on top of OpenShift Container Platform (converged) or alongside OpenShift Container Platform (independent). OpenShihft Container Storage is not tied down to a single availability zone or node making it very attractive for surviving an outage. Complete instructions for using RHOCS can be found in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/3.11/html-single/deployment_guide/RHOCS[3.11 Deployment Guide].
```
Comment (kalexand-rh) [inline install_config/topics/red_hat_openshift_container_storage.adoc]: I'd replace lines 10 and later with something like this:

[id='about-RHOCS-{context}']
=  About Red Hat OpenShift Container Storage 
Red Hat OpenShift Container Storage (RHOCS) is a provider of agnostic persistent storage for {product-title} either in-house or in hybrid clouds. As a Red Hat storage solution, RHOCS is completely integrated with {product-title} for deployment, management, and monitoring regardless if it is installed on {product-title} (converged) or with {product-title} (independent). OpenShift Container Storage is not limited to a single availability zone or node, which makes it likely to survive an outage. You can find complete instructions for using RHOCS in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/3.11/html-single/deployment_guide/RHOCS[3.11 Deployment Guide].

That link doesn't work for me.

**Review by kalexand-rh — COMMENTED** [review]
@cooktheryan, PTAL?

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL? I know we're avoiding mixing modules with unmodularized content, but this might make more sense than copy/pasting the section.

**Comment by kalexand-rh** [comment]
Remember that 3.10 needs this link:
```
https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/3.10/html-single/deployment_guide/RHOCS[3.10 Deployment Guide].
```

**Comment by cooktheryan** [comment]
Thanks @kalexand-rh for the review

@kalexand-rh++

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL

**Comment by mbearer** [comment]
looks good....slight editorial suggestion to this text (take it or leave it assuming still accurate):
> After tagging any resources, restart the master services on the master and the node service on all nodes.

To:

After tagging any resources, restart the master services on the master and restart the node service on all nodes.

**Comment by kalexand-rh** [comment]
Thanks @mbearer!  I think it's still accurate. @cooktheryan, are you up for one more edit-and-squash before I merge?

**Comment by cooktheryan** [comment]
@kalexand-rh ready to go

**Comment by kalexand-rh** [comment]
Thanks @cooktheryan! I'll merge whenever Travis finishes. :)

**Pull request merged**